README.md

```diff
@@ -4,20 +4,20 @@
 
 The fairlearn project seeks to enable anyone involved in the development of artificial intelligence (AI) systems to assess their system's fairness and mitigate the observed unfairness. The fairlearn repository contains a Python package and Jupyter notebooks with the examples of usage.
 
-- [Current release](#current-release)
-- [What we mean by _fairness_](#what-we-mean-by-fairness)
-- [Overview of fairlearn](#overview-of-fairlearn)
-- [Install fairlearn](#install-fairlearn)
-- [Usage](#usage)
-- [Contributing](#contributing)
-- [Maintainers](#maintainers)
-- [Issues](#issues)
+- [Current release](https://github.com/fairlearn/fairlearn/tree/v0.4.0#current-release)
```

Comment:
Expanding these URLs will break the script I added to the release process.

Reply:
Yeah that was because I ran that script and didn't realize it had overwritten the file. Undid all of these